### PR TITLE
Orm implemented

### LINKED ISSUE #27: Build ORM
Build an ORM to interact with a relational database

### PR DESCRIPTION
**Pull request by Sddilora**
closes #27, #28

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 EMAIL = "sumeyyedilaradogan@gmail.com"
 AUTHOR = "Sddilora"
 REQUIRES_PYTHON = ">=3.6.0"
-VERSION = "1.0.2"
+VERSION = "1.0.3"
```
Comment (log101) [inline setup.py]: This seems like a minor release as per [Semantic Versioning](https://semver.org/). You might want to upgrade the version to 1.1.0?

**Review by log101 — CHANGES_REQUESTED** [review]
The version number should be changed

File: setup.py (hunk 1 above)
Comment (Sddilora) [reply setup.py]: I was planning to make this change after adding the usage of ORM to the documentation and implementing the rendering of MD files.

Comment (log101) [reply setup.py]: Implementing the MD templating would make another minor version if you ask me .) If you're planning to update the documentation as well, we could merge this pr after you made that change. Or upgrade to 1.1.1 after documentation is added.

Comment (Sddilora) [reply setup.py]: ok I will handle that. Thanks for your suggestion.

**Review by log101 — APPROVED** [review]

**Pull request merged**